apps/studio/components/interfaces/Organization/InvoicesSettings/InvoicesSettings.tsx

```diff
@@ -61,6 +68,27 @@ const InvoicesSettings = () => {
     )
   }
 
+  if (
+    selectedOrganization?.managed_by !== undefined &&
+    selectedOrganization?.managed_by !== 'supabase'
+  ) {
+    // TODO: get vercel invoices link
+
+    return (
+      <ScaffoldContainerLegacy>
+        <PartnerManagedResource
+          partner={selectedOrganization?.managed_by}
+          resource="Invoices"
+          ctaUrl={
+            selectedOrganization?.managed_by === 'vercel-marketplace'
+              ? `https://vercel.com/${'supabase'}/~/integrations/products/${process.env.NEXT_PUBLIC_VERCEL_MARKETPLACE_INTEGRATION_NAME}/${selectedOrganization.partner_id}/invoices`
```

Comment:
nit: unnecessary interpolation although im not sure if you guys intended the interpolation here based on the above comment?

```suggestion
              ? `https://vercel.com/supabase/~/integrations/products/${process.env.NEXT_PUBLIC_VERCEL_MARKETPLACE_INTEGRATION_NAME}/${selectedOrganization.partner_id}/invoices`
```